### ISSUE TITLE
No documentation of "trusted image/pdf" features

### ISSUE THREAD

**Issue by Ark-of-the-copepod** [QubesOS/qubes-issues#6574]
**Qubes OS version**
4

**Affected component(s) or functionality**

Documentation

**Brief summary**

I can't find documentation of "trusted images". I fail to bring up useful hits on DDG, e.g. searches return no useful results:
- "qubes" + "trusted image"
- "trusted image" site:https://www.qubes-os.org
- "trusted pdf" site:https://www.qubes-os.org

**How Reproducible**

n/a

**To Reproduce**

1. consult documentation
2. web search
3. 

**Expected behavior**

There should be entries in official documentation describing behavior of these very useful tools, including what programs are used, what transformations are performed on the files.

Information should address both:
1) safety to the user's computer
2) the anonymity provided (e.g. metadata stripping/spoofing/padding).

Conversely, if this information is already extant, it should be indexed to be discoverable with search methods used above.

**Actual behavior**

**Screenshots**

**Additional context**

**Solutions you've tried**

**Relevant [documentation](https://www.qubes-os.org/doc/) you've consulted**

**Related, [non-duplicate](https://www.qubes-os.org/doc/reporting-bugs/#new-issues-should-not-be-duplicates-of-existing-issues) issues**

**Comment by ctrlaltf24** [comment]
The first result on duckduckgo for `"Qubes" trusted pdf` yields the original blog post detailing the new feature in layman terms https://blog.invisiblethings.org/2013/02/21/converting-untrusted-pdfs-into-trusted.html , additionally the readme of the repo itself https://github.com/QubesOS/qubes-app-linux-pdf-converter has a brief description. However it was missing from the docs for some of the other CLIs here https://dev.qubes-os.org/projects/core-admin-client/en/latest/ the omission appears to be intentional as that page only details CLI tools that act as a client to qubes-core-admin. There should be another entry in https://www.qubes-os.org/doc/tools/ under the DomU section describing qvm-convert-pdf and qvm-connect-img as described in the blog post and the readme. I can open a PR later today with those additions, however it may take some time to be merged into the documentation as they need to be reviewed by the core team for accuracy.

**Comment by Ark-of-the-copepod** [comment]
8 year old posts from dormant blogs are not always helpful. They can be dated and advise things that are ineffective, confusing or dangerous. That post was nicely written but long and was written during Qubes 1.0. I skipped it on purpose. This shouldn't be relied on for users, especially 'new entrants'.

The Github page is good but brief. I did not think to look there. However, search returns pdf-converter with "pdf" but no results for "image".

In any case. Shouldn't official documentation be the first and best source of information for users? One and only one unambiguous source of information.

Thanks for the new documentation efforts on this.

**Comment by Ark-of-the-copepod** [comment]
To be clear on what I would find helpful:
-

- explanation of the principle (short)
- quick reference: a table comparison of properties, pdf-untrusted v. pdf-trusted, jpg-untrusted v. image-trusted
- longer explanation, e.g. original post, reviewed.

**Comment by ctrlaltf24** [comment]
I have made a README for qvm-convert-img and submitted a PR to add those two READMEs to the official documentation. If you want to look at the code that handles the conversion process, take a look here https://github.com/QubesOS/qubes-linux-utils/blob/c16c721c8c8fb1c006f1e9c54ec93b1e2ae4b256/imgconverter/qubesimgconverter/__init__.py#L187

>     * explanation of the principle (short)
The READMEs offer this
>     * quick reference: a table comparison of properties, pdf-untrusted v. pdf-trusted, jpg-untrusted v. image-trusted
"Trusted" simply means reconstructed from the raw RGBA data. If you want to take a look, I encourage you to take a look at what changes between a normal image and a "Trusted" one. You can observe metadata using ImageMagick's `identify -verbose ./image.jpg`, or if you want to get fancy `diff <(identify -verbose ./trusted.jpg) <(identify -verbose ./untrusted.jpg)`. Simply put the image headers are not considered when reconstructing the image from RGBA data, resulting in some metadata loss.
>     * longer explanation, e.g. original post, reviewed.
The original post already details the mechanisms quite well, and the code is there if you want to see how it works under the hood.

**Comment by Ark-of-the-copepod** [comment]
Good, thank you. I can't find the table becuase I'm not familiar with GitHub, but the words seem good.

I still think these functions should be documented in https://www.qubes-os.org/doc/, not just under the DomU command line functions https://www.qubes-os.org/doc/tools/. These functions are available under the right-click menu options in nautilus. That's how I found them but couldn't find documentation on them. I never would have looked at command line functions, or GitHub.

**Comment by ctrlaltf24** [comment]
Searching for "Convert to Trsuted PDF Qubes" on ddg results in the blog post explaining the mechanism in Qubes 1, then the Github repo with the added README. There is sufficient documentation that a dedicated user can find. If you think otherwise, PRs are welcome on the docs if you want to add a section for natuilus right click menus like the command line tools. In my eyes this issue is solved (once the PRs are merged).
